package.json

```diff
@@ -16,7 +16,8 @@
     "prettier:check": "prettier --ignore-path .gitignore --check \"**/*{.ts,.tsx,.js,.jsx,.css,.json}\"",
     "start-jupyter": "mkdir -p build/cypress-tests && jupyter lab --config=./tests/test-config.py",
     "test": "start-server-and-test start-jupyter http-get://localhost:8888?token=test cy:run",
-    "test-debug": "start-server-and-test start-jupyter http-get://localhost:8888?token=test cy:open"
+    "test-debug": "start-server-and-test start-jupyter http-get://localhost:8888?token=test cy:open",
+    "test-integration": "lerna run test --scope \"@elyra/*\" --concurrency 1 --stream"
```

Comment:
FYI the cypress tests are integration tests, these new jest tests are unit tests. So we should probably rename the current calls and name the new ones `test` instead